CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+3.1.0 (in progress)
+===================
+
+New Features
+------------
+
+- Removed workaround for this Chrome [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=774303).
+  Now, we expose RTCPeerConnection's native `ontrack` event.
```

Comment:
Nitpick: rather than "native `ontrack` event", why not "native RTCTrackEvent"? Also, I guess we could spin this as a new feature, because that native RTCTrackEvent will probably include, e.g., the RTCRtpSender and other values we didn't copy over to our "fake" RTCTrackEvent.